### PR TITLE
Adding the SqlClient Stress Test execution engine

### PR DESCRIPTION
**Pull request by saurabh500**
This pull request adds the SqlClient Reliability test execution engine for .Net core. 
The tests to be executed with this engine will follow.

### CODE REVIEW

**Comment by saurabh500** [comment]
cc @corivera @geleems

**Comment by corivera** [comment]
Can we change the folder name from "stress" to "StressTests"? The other StressTest folder could then be renamed to something like "StressHarness" since there aren't any tests cases in that folder (and all the files in there are in the DPStressHarness namespace, anyway).

File: src/System.Data.SqlClient/tests/stress/StressTest/MemApi.Windows.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿
```
Comment (corivera) [inline src/System.Data.SqlClient/tests/stress/StressTest/MemApi.Windows.cs]: Needs a copyright header

File: src/System.Data.SqlClient/tests/stress/StressTest/Constants.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DPStressHarness
+{
+    static public class Constants
```
Comment (corivera) [inline src/System.Data.SqlClient/tests/stress/StressTest/Constants.cs]: Should be "public static". There are also some other instances of static being in front of things.

**Comment by corivera** [comment]
LGTM after other comments.

**Pull request merged**